contracts/cw-proposal-single/src/error.rs

```diff
@@ -31,9 +31,6 @@ pub enum ContractError {
     #[error("Proposal is ({size}) bytes, must be <= ({max}) bytes")]
     ProposalTooLarge { size: u64, max: u64 },
 
-    #[error("Proposal is not open ({id})")]
-    NotOpen { id: u64 },
-
     #[error("Proposal is expired ({id})")]
```

Comment:
nit: can we make the message #[error("Proposal ({id}) is expired")]

Reply:
ok, changed this error